studio/components/interfaces/Organization/Documents/SOC2.tsx

```diff
@@ -0,0 +1,133 @@
+import Link from 'next/link'
+import { useParams } from 'common'
+import {
+  ScaffoldSection,
+  ScaffoldSectionContent,
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import AlertError from 'components/ui/AlertError'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { useOrgSubscriptionQuery } from 'data/subscriptions/org-subscription-query'
+import { Button, IconDownload, Modal } from 'ui'
+import { useState } from 'react'
+import { useStore } from 'hooks'
+import ConfirmationModal from 'components/ui/ConfirmationModal'
+import { getDocument } from 'data/documents/document-query'
+
+const SOC2 = () => {
+  const { slug } = useParams()
+  const { ui } = useStore()
+  const {
+    data: subscription,
+    error,
+    isLoading,
+    isError,
+    isSuccess,
+  } = useOrgSubscriptionQuery({ orgSlug: slug })
+  const currentPlan = subscription?.plan
+  const [isOpen, setIsOpen] = useState(false)
+
+  const fetchSOC2 = async (orgSlug: string) => {
+    try {
+      const soc2Link = await getDocument({ orgSlug, docType: 'soc2-type-2-report' })
+      if (soc2Link?.fileUrl) window.open(soc2Link.fileUrl, '_blank')
+      setIsOpen(false)
+    } catch (error: any) {
+      ui.setNotification({
+        category: 'error',
+        message: `Failed to download SOC2 report: ${error.message}`,
+      })
+    }
+  }
+
+  return (
+    <ScaffoldSection>
+      <ScaffoldSectionDetail className="sticky space-y-6 top-12">
+        <p className="text-base m-0">SOC2 Type 2</p>
+        <div className="space-y-2 text-sm text-foreground-light m-0">
+          <p>
+            For organizations on Teams plan or above, you have access to our most recent SOC2 Type 2
```

Comment:
Organizations on the Teams plan or above have access to our most recent SOC2 Type 2 report.  (?)